.github/workflows/conda-package.yml

```
@@ -194,6 +206,7 @@ jobs:
         run: python -c "import dpnp, dpctl, numba_dpex; dpctl.lsplatform()"
 
       - name: Run tests
+        if: ${{matrix.run_gdb  == false}}
```

```suggestion
        if: ${{matrix.run_gdb == false}}
```
Please try to be consistent with spaces. Some for other PRs and places

---

Does `if: !${{matrix.run_gdb}}` work?

---

no

---

moved ` !`  inside, not sure that will work